src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDown.groovy

```diff
@@ -23,7 +23,7 @@ class ComposeDown extends DefaultTask {
             if (extension.removeContainers) {
                 if (getDockerComposeVersion() >= VersionNumber.parse('1.6.0')) {
                     project.exec { ExecSpec e ->
-                        e.commandLine extension.composeCommand('down')
+                        e.commandLine extension.composeCommand('down', '--rmi', 'local', '--volumes')

```

Comment:
Good point. According to the documentation it does this:
> remove only images that don't have an custom name set by the `image` field

So it actually removes images that was built during `docker-compose up` and doesn't remove images pulled from Registry. So if you use for example `cassandra` from image (which is usual) then the plugin doesn't remove it and  so the image isn't pulled in next runs.